boxfile.yml

```diff
@@ -105,6 +113,20 @@ worker.sidekiq:
     data.storage:
       - public/system
 
+
+worker.cron_only:
+  start: sleep 365d
+
+  writable_dirs:
+    - tmp
+
+  log_watch:
+    rake: 'log/production.log'
+
+  network_dirs:
+    data.storage:
+      - public/system
+
   cron:
     - id: generate_static_gifs
```

Comment:
Wait, hold on. Are these run as cron? They are not regular tasks.. They were upgrade one-offs...

Reply:
I didn't see any mention of _when_ to run them - or not - so they've been hiding in the background for a while, now. Will happily remove whichever ones I can! It's pretty clear you at least mean the task to generate a static image from a gif; does this apply to any of the others as well?